dsew_community_profile/Makefile

```diff
@@ -27,4 +27,3 @@ test:
 
 clean:
 	rm -rf env
-	rm -f params.json
```

Comment:
What's the reason for this change? Stale params files have caused problems before.

Reply:
I see, I will remove it.